src/Controllers/DatabaseController.js

```diff
@@ -2,13 +2,28 @@
 // Parse database.
 
 import intersect from 'intersect';
+import _         from 'lodash';
 
 var mongodb = require('mongodb');
 var Parse = require('parse/node').Parse;
 
 var SchemaController = require('../Controllers/SchemaController');
 const deepcopy = require('deepcopy');
 
+function addWriteACL(query, acl) {
```

Comment:
name is somehow misleading, as we do a deepCopy and return a new object, but I believe that's ok.